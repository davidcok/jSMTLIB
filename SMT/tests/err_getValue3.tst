; get-value after an assertion set command
(set-option :produce-models true)
(set-logic QF_UF)
(declare-fun x () Bool)
(assert true)
(check-sat) ; sat
(push 1)
(get-value (x)) ; invalid because of the push
