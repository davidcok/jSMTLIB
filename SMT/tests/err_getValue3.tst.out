success
success
success
success
unknown
success
(get-value (x)) ; invalid because of the push
^^^^^^^^^^^^^^^
(error "A get-value command is valid only after check-sat has returned sat or unknown")
